sys/net/gnrc/sock/include/gnrc_sock_internal.h

```diff
@@ -37,6 +37,22 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default port range for dynamic source port allocation
+ */
+#define GNRC_SOCK_DYN_PORTRANGE_MIN (49152U)
+#define GNRC_SOCK_DYN_PORTRANGE_MAX (65535U)
```

Comment:
The range of dynamic port numbers are defined by the [IANA](http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml).